src/components/templates/TemplateInventory.vue

```diff
@@ -280,19 +285,15 @@ export default {
             this.$emit('updateRequest', spotData);
         },
 
-        updateLat(spotData, lat) {
+        updateLatLng(spotData, latlng) {
+            const coordinate = latlng.split(',');
```

Comment:
Create a util method for extracting lat lng from string and use the same in SRP to parse latlng for query param

Reply:
done